app/styles/ui/_commit-message-avatar.scss

```diff
@@ -17,10 +17,10 @@
 
     border-radius: 50%;
     margin-right: var(--spacing-half);
+  }
 
-    .avatar {
-      margin-right: 0;
-    }
+  .toggletip {
+    margin-right: var(--spacing-half);
```

Comment:
This is just a fix because the outline of the avatar right next to the `Summary` text field, which was also picking the right margin we added to the avatar image 😳